src/psyclone/tests/psyGen_test.py

```diff
@@ -54,7 +54,8 @@
 from psyclone.psyGen import TransInfo, Transformation, PSyFactory, NameSpace, \
     NameSpaceFactory, OMPParallelDoDirective, PSy, \
     OMPParallelDirective, OMPDoDirective, OMPDirective, Directive, CodeBlock, \
-    Assignment, Reference, BinaryOperation, Array, Literal, Node, IfBlock
+    Assignment, Reference, BinaryOperation, Array, Literal, Node, IfBlock, \
+    BinaryOperation, KernelSchedule, Symbol, SymbolTable
```

Comment:
You have `BinaryOperation` twice in this list :-)

Reply:
Fixed